### ISSUE TITLE
Building packages fails with non-existent target

### ISSUE THREAD

**Issue by lissom** [quickfix/quickfix#421]
make dist (other dist commands too)
make  dist-gzip am__post_remove_distdir='@:'
make[1]: Entering directory `/h/cpage/git/quickfix'
make[1]: *** No rule to make target `README', needed by `distdir'.  Stop.
make[1]: Leaving directory `/h/cpage/git/quickfix'
make: *** [dist] Error 2

./configure --with-python3 --with-mysql --with-openssl=/usr --with-boost=$BOOST_VER --prefix $PREFIX

/usr/bin/install: cannot create regular file ‘/opt/python/3.11.2/lib/python3.11/site-packages/quickfix.py’: Permission denied

Python doesn't seem to honor --prefix so I also can't just dump everything somewhere and package that up.

Is there some other way this is supposed to be packaged up?

**Comment by lissom** [comment]
It looks like, for the python:
`make install DESTDIR=$PREFIX`
is the solution.  Would you please confirm that is the correct way to build this?
Where 
`./configure --prefix=$PREFIX --python3 ...`
sets the path for the rest of the files.

**Comment by orenmnero** [comment]
Yeah, I'm not really sure what the best practice is here. --prefix doesn't really make sense for python as it needs a valid site-packages to place quickfix, which means it needs a python environment.

If you run while under a virtual environment it should install there, otherwise maybe we can add a path parameter to --with-python3 where you can specify a python environment. 

Open to other ideas, or information about best practices.
